mtp_noms_ops/apps/security/views/check.py

```diff
@@ -66,16 +67,72 @@ def get_context_data(self, **kwargs):
         return context
 
 
-#  class AutoAcceptRuleDetailView(SecurityDetailView):
-    #  """
-    #  View history of all auto-accept rules
-    #  """
-    #  title = gettext_lazy('Auto Accept')
-    #  list_title = gettext_lazy('Auto Accepts')
-    #  #  template_name = 'security/auto_accept_rule.html'
-    #  object_context_key = 'auto_accept_rule'
-    #  id_kwarg_name = 'auto_accept_rule_id'
-    #  list_url = reverse_lazy('security:auto_accept_rule_list')
+class AutoAcceptRuleDetailView(SimpleSecurityDetailView, FormView):
+    """
+    View history of all auto-accept rules
+    """
+    list_title = gettext_lazy('Auto Accepts')
+    template_name = 'security/auto_accept_rule.html'
+    object_context_key = 'auto_accept_rule'
+    id_kwarg_name = 'auto_accept_rule_id'
+    list_url = reverse_lazy('security:auto_accept_rule_list')
+    form_class = AutoAcceptDetailForm
+
+    def get_form_kwargs(self):
+        return dict(super().get_form_kwargs(), request=self.request, object_id=self.kwargs[self.id_kwarg_name])
+
+    def get_object_request_params(self):
+        return {
+            'url': f'/security/checks/auto-accept/{self.kwargs[self.id_kwarg_name]}/'
+        }
+        return super().get_object_request_params()
```

Comment:
tiny thingy: this line is unreachable (and would throw NotImplementedError)

Reply:
:+1: That was a vim typo. will fix